app/models/book.rb

```diff
@@ -1,4 +1,6 @@
 class Book < ActiveRecord::Base
+  SORT_CRITERIA_ADDED = "added"
```

Comment:
I introduced this to make things less fragile. A typo in a string can be easily missed, but a typo in a constant name will make Ruby fail loudly.